src/cmd-buildextend-live

```diff
@@ -556,8 +556,12 @@ boot
             # Note: virt-make-fs lets us do this as non-root
             efibootfile = os.path.join(tmpisoimages, 'efiboot.img')
             os.environ["LIBGUESTFS_BACKEND"] = "direct"
-            run_verbose(['virt-make-fs', '--type=vfat', efitarfile.name,
-                         efibootfile])
+            # On RHEL 8, when booting from a disk device (rather than a CD),
+            # https://github.com/systemd/systemd/issues/14408 causes the
+            # hybrid ESP to race with the ISO9660 filesystem for the
+            # /dev/disk/by-label symlink unless the ESP has its own label.
+            run_verbose(['virt-make-fs', '--type=vfat', '--label=EFI-SYSTEM',
```

Comment:
Optional: This fact is obvious if you're reading critically, but the extra comment in the suggestion below helps me comprehend when my brain is running in autopilot mode.

```suggestion
            # /dev/disk/by-label symlink unless the ESP has its own label.
            # We use --label=EFI-SYSTEM to avoid that problem.
            run_verbose(['virt-make-fs', '--type=vfat', '--label=EFI-SYSTEM',
```

Reply:
I also wonder if we should mention that this isn't an issue on Fedora (and probably RHEL9 but it's still not a bad idea to add the filesystem label there too. i.e. we'll probably never drop this label.